Update per-monitor-awareness-extenders.md (#3069)
MicrosoftDocs/visualstudio-docs: docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md

## Enabling PMA
To enable PMA in Visual Studio, the following requirements need to be met:
1)  Windows 10 April 2018 Update (v1803, RS4) or later
2)  .NET Framework 4.8 RTM or greater
3)  Visual Studio 2019 with the ["Optimize rendering for screens with different pixel densities"](https://docs.microsoft.com/visualstudio/ide/reference/general-environment-options-dialog-box?view=vs-2019) option enabled

Once these requirements are met, Visual Studio will automatically enable PMA mode across the process.

> **Note:** Windows Forms content in VS ( for ex: Property Browser) will support PMA only when you have Visual Studio 2019 Update #1.

Reviewer: ```suggestion
**Note:** Windows Forms content in VS (for example Property Browser) will support PMA only when you have Visual Studio 2019 Update #1.
```

> **Note:** Windows Forms content in VS (for example Property Browser) will support PMA only when you have Visual Studio 2019 Update #1.

Reviewer 2: ```suggestion
> [!NOTE]
> Windows Forms content in VS (for example Property Browser) will support PMA only when you have Visual Studio 2019 Update #1.
```

#### Out-of-process UI
Some UI is created out-of-process and if the creating external process is in a different DPI awareness mode than Visual Studio, this can introduce any of the previous rendering issues.

#### Windows Forms controls, images or layouts rendered incorrectly

> Not all of the Windows Forms content support PMA mode. As a result, you may see rendering issue with incorrect layouts or scaling. A possible solution here is,  force Windows Forms content to be rendered in "System Aware"  DpiAwarenessContext ( refer to [Forcing a control into a specific DpiAwarenessContext](https://github.com/MicrosoftDocs/visualstudio-docs-pr/blob/master/docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md#forcing-a-control-into-a-specific-dpiawarenesscontext)).

Reviewer 2: "visualstudio-docs-pr" isn't a public repo, so non-MS people won't be able to see this file

> Not all of the Windows Forms content support PMA mode. As a result, you may see rendering issue with incorrect layouts or scaling. A possible solution here is,  force Windows Forms content to be rendered in "System Aware"  DpiAwarenessContext ( refer to [Forcing a control into a specific DpiAwarenessContext](https://github.com/MicrosoftDocs/visualstudio-docs-pr/blob/master/docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md#forcing-a-control-into-a-specific-dpiawarenesscontext)).

Reviewer: ```suggestion
Not all of the Windows Forms content support PMA mode. As a result, you may see rendering issue with incorrect layouts or scaling. A possible solution in this case is to explicitly render Windows Forms content as `SystemAware`. You can find how to set `DpiAwarenessContext` to `SystemAware`  in [Forcing a control into a specific DpiAwarenessContext](https://github.com/MicrosoftDocs/visualstudio-docs-pr/blob/master/docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md#forcing-a-control-into-a-specific-dpiawarenesscontext) paragraph of this article.
```

#### Windows Forms controls or windows not displaying
One of the main causes for this issue is developers trying to reparent a control or window with one DpiAwarenessContext to a window with a different DpiAwarenessContext.

The following pictures show the current **default** Windows operating system restrictions in parenting windows:

![A screenshot of the correct parenting behavior](../../extensibility/ux-guidelines/media/PMA-parenting-behavior.PNG)

> **Note:** you can change this behaviour by setting the Thread Hosting Behaviour ( refer to [DpiHostinBehaviour](https://docs.microsoft.com/en-us/windows/desktop/api/windef/ne-windef-dpi_hosting_behavior) )

Reviewer 2: Docs links shouldn't have the locale in them
```suggestion
**Note:** you can change this behaviour by setting the Thread Hosting Behaviour ( refer to [DpiHostinBehaviour](https://docs.microsoft.com/windows/desktop/api/windef/ne-windef-dpi_hosting_behavior) )
```

Reviewer 2: Might also make sense to use `[!NOTE]` alert section too
